### DISCUSSION TITLE
Using a GitHub checkout of Taiko

### DISCUSSION THREAD

**Discussion by fmeum** [getgauge/taiko#2094] [Development]
I would like to use a GitHub checkout of Taiko (e.g., the `v1.2.5` tag) for my tests so that I can add some more debug prints to failing selectors. However, if I reference a GitHub release of Taiko in my `package.json`, e.g. `getgauge/taiko#v1.2.5`, suddenly all my tests fail already on the first selector action.

What is the preferred way to use a local checkout of Taiko in another project?

**Comment by sriv** [comment]
since taiko is an npm module, you should be able to use `npm install <path to local taiko>` in your project. Would that help? 

It will make changes to your package.json which you may want to review before any commits.

ref: https://docs.npmjs.com/cli/v7/commands/npm-install
